episodes/13-synchronization.md

```diff
@@ -177,22 +173,23 @@ this is new task working: 9
 this is new task working: 10
 New task finished
 and now it is done
-~~~
-{: .output}
+```
 
-> ## Discussion
->
-> What would happen if we assign a value to _x_ right before launching the new
-> task? What would happen if we assign a value to _x_ right before launching
-> the new task and after the _writeln("and now it is done");_ statement?
->
-> Discuss your observations.
-{: .discussion}
+::::::::::::::::::::::::::::::::::::::: discussion
+
+## Discussion
```

Comment:
Include notes (or Solutions) for the Instructor in Discussion blocks like this.

Reply:
I added a paragraph explaining how the sync variable works in this example. I also changed the discussion with an example that will block for sure (the previous one did not always block), providing an explanation (solution) of why it blocks. Resolved through a commit.